spec/lib/diaspora_federation/entities/relayable_spec.rb

```diff
@@ -26,8 +26,8 @@ module DiasporaFederation
 
     describe "#verify_signatures" do
       it "doesn't raise anything if correct signatures with legacy-string were passed" do
```

Comment:
You're right, and this test didn't even do what it's description said ;) But since we don't differentiate new or old in `#verify_signature` (we just need to respect the order), I changed the tests to test that, and removed the `new signatures` part, because it was obsolete now.